Persistence/Entities/PostMachine.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace shipping_service.Persistence.Entities
+{
+    public class PostMachine
+    {
+        public ulong Id { get; set; }
+        public string Name { get; set; }
+        public DateTime Created { get; set; }
+        public DateTime Modified { get; set; }
+        public string Address { get; set; }
+        public ICollection<Package> PackagesWithThisSource { get; set; }
```

Comment:
This along with PackagesWithThisDestination is not necessary. Just create `public IEnumerable<Package> Packages {get; set; }`, if we need, we can tell which is source and which is destination by accessing the packages and checking their machines.

Reply:
Not necessary, but I think it will be useful. We have this information anyway in the package (both source and destination), so why not expose it here too?

Reply:
Well, by the db diagram this is correct. Althought I am not sure where this will be useful.